src/sif/autocost.cc

```diff
@@ -76,6 +77,10 @@ constexpr float kLeftSideTurnCosts[] = {kTCStraight,         kTCSlight,  kTCUnfa
 constexpr float kMinFactor = 0.1f;
 constexpr float kMaxFactor = 100000.0f;
 
+// min and max factors to apply when use tracks
+constexpr float kMinTrackFactor = 0.8f;
+constexpr float kMaxTrackFactor = 10.f;
```

Comment:
I used the same factors as for alleys https://github.com/mapbox/api-valhalla/blob/589913ce8df30a6a8948df026d10ccdd3e0b80b1/src/lib/buildDirectionsDrivingRequest.js#L58 .